ansible/roles/jitsi-videobridge/tasks/install.yml

```
@@ -131,23 +131,24 @@
     src: "jvb-stats.sh"
     mode: 0755
     owner: root
-  when: jvb_cloud_provider == "aws"
+  when: jvb_cloud_provider == "aws" and jvb_enable_stats_cron
 
 - name: JVB stats script upload oracle
   ansible.builtin.copy:
     dest: "{{ path_to_stats_script }}"
     src: "jvb-stats-oracle.sh"
     mode: 0755
     owner: root
-  when: jvb_cloud_provider == "oracle"
+  when: jvb_cloud_provider == "oracle" and jvb_enable_stats_cron
```

I believe this is desired during install even if jvb_enable_stats_cron is not on, since the install part of the role is used during image build time.
```suggestion
  when: jvb_cloud_provider == "oracle"
```